virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -14,39 +14,26 @@ The Scheduling, Scale, and Performance (SSP) Operator integrates {VirtProductNam
 * Run commands in VMs
 * Manipulate disk images with `libguestfs` tools
 
-:FeatureName: Managing virtual machines with {pipelines-title}
-include::snippets/technology-preview.adoc[]
-
-
 [id="prerequisites_virt-managing-vms-openshift-pipelines"]
 == Prerequisites
 
 * You have access to an {product-title} cluster with `cluster-admin` permissions.
 * You have installed the OpenShift CLI (`oc`).
 * You have link:https://docs.openshift.com/pipelines/latest/install_config/installing-pipelines.html[installed {pipelines-shortname}].
 
-
-include::modules/virt-deploying-ssp.adoc[leveloffset=+1]
-
 include::modules/virt-supported-ssp-tasks.adoc[leveloffset=+1]
 
-
 [id="example-pipelines_virt-managing-vms-openshift-pipelines"]
-== Example pipelines
-
-The SSP Operator includes the following example `Pipeline` manifests. You can run the example pipelines by using the web console or CLI.
-
-You might have to run more than one installer pipeline if you need multiple versions of Windows. If you run more than one installer pipeline, each one requires unique parameters, such as the `autounattend` config map and base image name. For example, if you need Windows 10 and Windows 11 or Windows Server 2022 images, you have to run both the Windows efi installer pipeline and the Windows bios installer pipeline. However, if you need Windows 11 and Windows Server 2022 images, you have to run only the Windows efi installer pipeline.
-
-Windows EFI installer pipeline:: This pipeline installs Windows 11 or Windows Server 2022 into a new data volume from a Windows installation image (ISO file). A custom answer file is used to run the installation process.
+[id="Windows-EFI-installer-pipeline-virt-managing-vms-openshift-pipelines"]
+== Windows EFI installer pipeline
 
-Windows BIOS installer pipeline:: This pipeline installs Windows 10 into a new data volume from a Windows installation image, also called an ISO file. A custom answer file is used to run the installation process.
+The SSP Operator includes the `Pipeline` manifest of the Windows EFI installer pipeline. You can run the pipeline by using the web console or CLI.
```

Comment:
The SSP Operator no longer deploys the pipelines. The pipelines and tasks are hosted in "hubs" - https://github.com/openshift-pipelines/tektoncd-catalog/tree/p (for now it is GH repo, but in Q4 this year, the tasks and pipelines will be available from catalog.redhat.com). From there user can install the tasks / pipelines

Reply:
@ksimon1 Please check it now.